packages/core/src/external/aws/lambda-logic/bundle-to-html-bmi.ts

```diff
@@ -1207,7 +1280,7 @@ function createMedicationSection(
 
   const recentMedications = medicationStatements.filter(medicationStatement => {
     return (
-      dayjs(medicationStatement.effectivePeriod?.start).isAfter(dayjs().subtract(3, "year")) ||
+      dayjs(medicationStatement.effectivePeriod?.start).isAfter(dayjs().subtract(4, "months")) ||
```

Comment:
just a thought:

this seems oddly specific. Do we want to add a note somewhere in the MR section, like `Medications (last 4 months)`

Reply:
I mean this is something they specifically wanted - once we add the MR summary engine it wont be an issue